sdk/provisioning/Azure.Provisioning/tests/Infrastructure/StorageBlobDefaults/resources/rg_TEST_module/rg_TEST_module.bicep

```diff
@@ -1,6 +1,6 @@
 
-resource storageAccount_rJRF56wLn 'Microsoft.Storage/storageAccounts@2022-09-01' = {
-  name: 'photoacctdbd29b86eb654ef'
+resource storageAccount_8QqXaM2M1 'Microsoft.Storage/storageAccounts@2022-09-01' = {
```

Comment:
does this name remain constant each time we generate?

unless you change the prefix of course

Reply:
No this will be fixed in separate PR